functions/exercise21/scriptSpec.js

```diff
@@ -0,0 +1,15 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getPossibleSubset*/
+
+describe("Getting all possible subset with a fixed length combinations in an array", function () {
+    it("If the array [1, 2, 3] we will get all subsets of length 2, we get subsets [2, 1], [3, 1], [3, 2], [3, 2, 1]", function () {
+        var subsets = getPossibleSubset([1, 2, 3], 2);
+        expect(subsets).toEqual(
+            [3, 2],
+            [3, 1],
+            [2, 1],
+            [3, 2, 1]
```

Comment:
subset [3,2,1] is not correct here as number of elements is 3, not 2. There is an error in excercise description.